### ISSUE TITLE
Realtime Line Animation

### ISSUE THREAD

**Issue by psumstr** [ecomfe/echarts#1847]
I'm building a real time line chart where the data changes every second and a new date gets added to the data array while the first point exists the array. Currently, I'm experiencing an animation that is distracting and hard to follow as described here:

http://bost.ocks.org/mike/path/

Do you have any suggestions on how I can achieve the animation where I can interpolate the transform of the path/line? I'm looking for the solution as described in the link above.

**Comment by Ovilia** [comment]
Does this example help? http://echarts.baidu.com/doc/example/dynamicLineBar.html

**Comment by psumstr** [comment]
Thanks for the reply. I have seen that example, but I wasn't sure if that works because it is using the addData method of the chart. I was trying to replace the full array of data in the series, so not sure if that's what's causing the issue.

**Comment by Ovilia** [comment]
I guess the main idea should be similar, since `addData` does nothing special than pushing a new element to the end of the array. You may post some data for me to understand your scenario better.
